src/bin/pg_autoctl/cli_common.c

```diff
@@ -949,17 +949,25 @@ set_first_pgctl(PostgresSetup *pgSetup)
 	}
 
 	/* then, use PATH and fetch the first entry there for the monitor */
-	if (!search_path_first("pg_ctl", pgSetup->pg_ctl))
+	if (search_path_first("pg_ctl", pgSetup->pg_ctl))
```

Comment:
If this fails, there's an error log like this coming from `search_path_first`, even though it's not a hard error:
```
10:26:42 1793 ERROR Failed to find pg_ctl command in your PATH
```

I think it makes sense to add a loglevel to `search_path_first`.

Reply:
Good point, done now.